src/overlays/actors/ovl_En_Zl2/z_en_zl2.c

```diff
@@ -27,265 +101,1664 @@ const ActorInit En_Zl2_InitVars = {
+        Matrix_Translate(-427.0f, -1.0f, 3.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 5, "../z_en_zl2.c", 1189);
+        Matrix_Get(&sp34);
+        func_800D20CC(&sp34, &sp2C, 0);
+        if (func_800C0D28(globalCtx) == 0) {
+            func_80B4EE38(this, sp2C.y, 18);
+            func_80B4F230(this, sp2C.x, 19);
+            func_80B4EF64(this, sp2C.z, 20);
+        }
+        Matrix_RotateRPY(unk_1DC[18] + kREG(49), unk_1DC[19] + kREG(50), unk_1DC[20] + kREG(51), MTXMODE_APPLY);
+        Matrix_Translate(-446.0f, -52.0f, -84.0f, MTXMODE_APPLY);
+        Matrix_ToMtx(sp74 + 6, "../z_en_zl2.c", 1208);
+        Matrix_Pull();
+        Matrix_Pull();
+        this->unk_24C = 1;
+    }
+    return 0;
+}
+
+void EnZl2_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx);
+#ifdef NON_MATCHING
+// Stack issue in two instructions - made much better with the pad
+void EnZl2_PostLimbDraw(GlobalContext* globalCtx, s32 limbIndex, Gfx** dList, Vec3s* rot, Actor* thisx, Gfx** gfx) {
+    s32 pad[2];
+    EnZl2* this = THIS;
+    Player* player;
+
+    if (limbIndex == 10) {
+        if ((this->unk_254 != 0) && (globalCtx->csCtx.frames >= 0x384)) {
+            gSPDisplayList(gfx[0]++, &D_0600BAE8);
+        }
+
+        player = PLAYER;
+        Matrix_Push();
+        if (player->unk_15D == 0xFF) {
+            Matrix_Put(&player->mf_A20);
+            Matrix_Translate(180.0f, 979.0f, -375.0f, MTXMODE_APPLY);
+            Matrix_RotateRPY(-0x5DE7, -0x53E9, 0x3333, MTXMODE_APPLY);
+            Matrix_Scale(1.2f, 1.2f, 1.2f, MTXMODE_APPLY);
+            gSPMatrix((*gfx)++, Matrix_NewMtx(globalCtx->state.gfxCtx, "../z_en_zl2.c", 1253),
+                      G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList((*gfx)++, &D_0600BAE8);
+        }
+        Matrix_Pull();
+    }
+}
+#else
+#pragma GLOBAL_ASM("asm/non_matchings/overlays/actors/ovl_En_Zl2/EnZl2_PostLimbDraw.s")
+#endif
+
+void func_80B4FCCC(EnZl2* this, GlobalContext* globalCtx) {
+    s32 unk_274 = this->unk_274;
+
+    gSegments[6] = PHYSICAL_TO_VIRTUAL(globalCtx->objectCtx.status[unk_274].segment);
+}
+
+void func_80B4FD00(EnZl2* this, AnimationHeader* animation, u8 arg2, f32 transitionRate, s32 arg4) {
+    f32 frameCount = SkelAnime_GetFrameCount(&animation->genericHeader);
+    f32 playbackSpeed;
+    f32 unk0;
+    f32 fc;
+
+    if (arg4 == 0) {
+        unk0 = 0.0f;
+        fc = frameCount;
+        playbackSpeed = 1.0f;
+    } else {
+        fc = 0.0f;
+        unk0 = frameCount;
+        playbackSpeed = -1.0f;
+    }
+
+    SkelAnime_ChangeAnim(&this->skelAnime, animation, playbackSpeed, unk0, fc, arg2, transitionRate);
+}
+
+void func_80B4FD90(EnZl2* this, GlobalContext* globalCtx) {
+    func_80B4FD00(this, &D_0600B5FC, 0, 0.0f, 0);
+    this->action = 1;
+}
+
+void func_80B4FDD4(EnZl2* this) {
+    if (func_800A56C8(&this->skelAnime, 14.0f)) {
+        func_80078914(&this->actor.unk_E4, 0x802);
+    }
```

Comment:
E4 was renamed to projectedPos in a recent pr, thats why your jenkins build failed